packages/ra-core/src/controller/edit/useEditController.spec.tsx

```diff
@@ -31,11 +28,15 @@ describe('useEditController', () => {
         const dataProvider = ({ getOne } as unknown) as DataProvider;
         render(
             <CoreAdminContext dataProvider={dataProvider}>
-                <SaveContextProvider value={saveContextValue}>
-                    <EditController {...defaultProps}>
-                        {({ record }) => <div>{record && record.title}</div>}
-                    </EditController>
-                </SaveContextProvider>
+                <EditController {...defaultProps}>
+                    {({
+                        record,
+                        save,
```

Comment:
why do you unstructure save, saving and the others if you don't use them? I don't see the purpose of this change. Same for the following tests.